agent/lib/src/utils.dart

```diff
@@ -649,3 +649,20 @@ Future<Null> unlockKeyChain() async {
         canFail: false, silent: true);
   }
 }
+
+Future<Null> cpFileToGcs(String file, String gsBucket) async {
+  if (Platform.isLinux || Platform.isMacOS) {
```

Comment:
Do we want to support windows?

Reply:
Windows would fall into the else clause which runs the command a different way. Is that a problem?

I'm going by the previous experiences of @keyonghan

Reply:
On the other hand, in another comment @keyonghan mentions that the previous PR upon which this is based failed on Windows agents...